ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -20,32 +20,38 @@ import { dateFormat } from 'core/helpers/date-format';
 
 export default class MessagesList extends Component {
   @service store;
+  @service router;
+  @service flashMessages;
 
-  get getMessages() {
+  get formattedMessages() {
     return this.args.messages.map((message) => {
-      let badgeDisplayText = '';
+      const badgeDisplay = {};
```

Comment:
Okay, I might be reading this wrong, but again it looks like we're modifying the this.args.messages? Two options, do this logic in the model. Or, instead let's not add this `badgeDisplay` object back on to the this.args.messages.message, but instead just return an array of these badeDisplays and use that in this component. Does that make sense?